lib/theme/toast_messages/hs_toast_theme.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/toast_messages/toast_messages.dart';
+
+class ToastMessageError extends StatelessWidget {
+  final String message;
+  final String title;
+  const ToastMessageError(
+      {super.key, required this.message, required this.title});
+  @override
+  Widget build(BuildContext context) {
+    // TODO: implement build
+    return ToastMessageContainer(
+        key: key,
+        backgroundColor: const Color.fromARGB(255, 255, 241, 241),
+        icon: SvgPicture.asset(Assets.images.error),
+        toastTitle: title,
+        toastContent: message);
+  }
+}
+
+class ToastMessageContainer extends StatelessWidget {
+  final Color backgroundColor;
+  final SvgPicture icon;
+  SvgPicture? closeIcon;
+  Key? key;
+  String toastTitle;
+  String toastContent;
+  ToastMessageContainer(
+      {super.key,
+      required this.backgroundColor,
+      required this.icon,
+      required this.toastTitle,
+      required this.toastContent,
+      this.closeIcon})
+      : assert(toastContent.length < 160,
```

Comment:
define a constant number for max toast message length

Reply:
fixed